### ISSUE TITLE
Question Regarding Performance Discrepancy with RAW-Adapter

### ISSUE THREAD

**Issue by ocean146** [cuiziteng/ECCV_RAW_Adapter#3]
Hi cuiziteng,

First of all, I want to express my appreciation for the impressive RAW-Adapter paper and thank you for providing such detailed and comprehensive code as an open-source resource. It's a fantastic contribution to the community!

However, I have encountered an issue while reproducing the results. My detection and segmentation metrics are significantly lower than the numbers reported in the paper. I followed the instructions in the README, using the same settings and launch commands. The only difference is that I used  NVIDIA 3090 instead of V100.

Could you please provide any suggestions or tips to help me align my results more closely with those reported in the paper? Any guidance or information would be greatly appreciated!
Thank you!

**Comment by cuiziteng** [comment]
Hi, thanks so much for your interests in our work !

Could you show more details, like which dataset? GPU number? what is the score you get compare with our pre-trained weights score ? 

Some potential reasons (in decreasing order of importance):

(0). Could you test directly using our provide pre-trained weights to see if the same results can be achieved?

(1). You mean "significantly lower", did you use pre-train weights in config? Are the comparison methods you run also significantly lower in our report ? 

(2). GPU number and GPU difference, GPU number is essential and should corresponding to learning rate. 

(3). There are some instabilities in training, you can experiment with training several times.

**Comment by ocean146** [comment]
Thanks for your kind response.

(0) We directly used the pre-trained weights for testing, and our results are consistent with those reported in the paper.

(1) We used the config files downloaded from your repository, with the only difference being the dataset path. We have run experiments for both detection and segmentation, but the results are significantly lower than those reported in the paper.

(2) We conducted our experiments on a single 4090 GPU. Should we adjust the learning rate settings in the original config files?

Thanks!

**Comment by cuiziteng** [comment]
Thanks for you detailed information. In our experiments, all SP-RCNN-related experiments for the detection task, as well as the entire segmentation task, were conducted using four GPUs.

<img width="1068" alt="image" src="https://github.com/user-attachments/assets/980f1922-50a5-4167-a8ab-394ba16e033e">

<img width="1098" alt="image" src="https://github.com/user-attachments/assets/de2cb603-1214-4179-b73a-5228b2b4f786">

If only a single GPU is available, it can be challenging to replicate the same results. We recommend reducing the learning rate proportionally, for example, to one-fourth of the original value. Additionally, for the segmentation task, the training duration is controlled by the number of iterations; if switching from four GPUs to one, it meas the training iters would become 1/4, so it’s advisable to extend the iterations by a factor of four. 

I believe these adjustments will significantly improve performance, although they still cannot guarantee identical results to those achieved with four GPUs.

**Comment by ocean146** [comment]
Thank you so much for your help. I have reproduced the results reported in the paper using four 4090 GPUs.
